suite/tests/CMakeLists.txt

```diff
@@ -3972,16 +3973,19 @@ if (BUILD_CLIENTS)
         set(core_sharded_dir
           "${PROJECT_SOURCE_DIR}/clients/drcachesim/tests/drmemtrace.threadsig-core-sharded.x64.tracedir")
         torunonly_simtool(core_on_disk ${ci_shared_app}
+          # Avoid the default core-sharded from re-scheduling the trace.
           "-indir ${core_sharded_dir} -tool basic_counts" "")
```

Comment:
Missing `-no_core_sharded`.

Or it's not needed because basic_counts doesn't prefer core-sharded? Please clarify in the new comment added above.

Reply:
Not needed.  Removed the new comment.